outputs.tf

```diff
@@ -13,15 +13,15 @@ output "namespaces" {
 
 output "rbac_group" {
   description = "The name of the Kubernetes RBAC group"
-  value       = var.enable_admin ? "system:masters" : var.name
```

Comment:
This is a destructive change. We should probably add another variable for "admin" role name and use that if provided else default to current behavior. @bryantbiggs wdyt?

Reply:
I fixed a value to replace the `system:masters` at first, but this will not give the ability of adding any other existing group or customizing the name. Maybe add one more condition?

Reply:
I think we need to think this change through a bit more and also include some examples to test and demonstrate the different usage variations. I'm trying to wrap my head around these changes and I still have a lot of questions:
- If I stamp out a role, can I make it both admin as well as namespace scoped - do I want that or do I want to enforce one or the other? 
- How do I create a custom admin that does not have the standard `system:masters` permissions group?
- Do I have to create the group externally and then pass that in to this module? Should this module create the group?

Reply:
* I think it should do both cluster-wide and namespace scopes, depending on the existing variables. I tried the way it is now, it's working for both scenarios using the `enable_admin`. However the output change will break stuff, so we can add a default there to replace the `system:masters`.
* We can give the interface to the user to choose, adding and existing externally created group or role (like system:masters), but it nothing is declared regarding that, the behavior should be not giving access to another user with `system:masters` other than the cluster creator.